synapseclient/core/upload/multipart_upload.py

```diff
@@ -80,6 +78,8 @@ def _executor(max_threads, shutdown_wait):
 
 
 class UploadAttempt:
+    """ """
```

Comment:
I personally like having the docstring with a new line myself. But, I'm not sure we have a defined standard around. this. This style guide shows them without: https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings

I would be ok with either, or both.

@thomasyu888 @jaymedina Thoughts on which we should use?


```suggestion
    """
    Used to handle multi-threaded operations for uploading one or parts of a file.
    """
```

Reply:
I have been following the convention below out of habit, but I can see how starting the first line under the first `"""` is visually cleaner, so I'm find with doing it this way from now on. Doesn't seem to go against pep257 rules.
```
def myfunc():
   """multi-line
   docstring

   """

Reply:
This class will need a docstring on it

Reply:
Lost this thread for a second. Made the commit, thanks